File: ARCADE/Intro/Level 7_3: absoluteValuesSumMinimization.cs
    using System;
     
    public class Test
    {
    	public static int absoluteValuesSumMinimization(int[] a){
            if (a.Length % 2 == 1) return a[a.Length / 2];
            else return a[a.Length / 2 - 1];
        }
    	public static void Main()
    	{
    		int [] a = new int []{2, 4, 7};
    		Console.WriteLine(absoluteValuesSumMinimization(a));
    	}
    }

File: ARCADE/Intro/Level 7_3: absoluteValuesSumMinimization.pas
program absoluteValuesSumMinimization;
 
Type
	ArrayInteger = Array of integer;
 
function absoluteValuesSumMinimization(a: ArrayInteger): integer;
begin
    if(length(a) mod 2 = 1) then absoluteValuesSumMinimization := a[length(a) div 2]
    else absoluteValuesSumMinimization := a[length(a) div 2 - 1];
end;
 
var a : ArrayInteger;
begin
	a := ArrayInteger.Create(2, 4, 7);
	writeln(absoluteValuesSumMinimization(a));
end.
